lib/pages/login_page.dart

```diff
@@ -1,10 +1,61 @@
 import 'package:flutter/material.dart';
+import 'package:school_erp/widgets/forgot_password.dart';
+import 'package:school_erp/widgets/login_body.dart';
+import 'package:school_erp/widgets/login_stack.dart'; // Import the custom container widget
```

Comment:
the common 'widgets' folder should only contain shared components

page specific widgets should go in their respective page folders:
pages/login/login_page.dart
pages/login/widgets/